presto-product-tests/src/main/resources/sql-tests/testcases/tpcds/q36.result

```diff
@@ -1,101 +1,101 @@
--- delimiter: |; types: DECIMAL|LONGNVARCHAR|CHAR|INTEGER|BIGINT
```

Comment:
Probably it was quarantined and so it was unable to test. it was left in this shape.